Ryujinx.Common/SystemInterop/UnixStream.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Ryujinx.Common.SystemInterop
+{
+    public partial class UnixStream : Stream, IDisposable
```

Comment:
```suggestion
    [UnsupportedOSPlatform("windows")]
    public partial class UnixStream : Stream, IDisposable
```
Just to make sure we never use it for Windows.

Reply:
sgtm